README.md

```diff
@@ -798,12 +798,12 @@ Developers in the project may not follow consistent code security practices, lea
 
 ## ![✔] 6.14. Limit payload size using a reverse-proxy or a middleware
 
-**TL;DR:** Here we will talk about the easiness of exhausting the system using large request payloads
+**TL;DR:** DOS attacks of many varieties can be mitigated by limiting the body size of incoming requests to your web application.
 
-**Otherwise:**
+**Otherwise:** requests with no restriction towards their payload/body size may simply cause a denial-of-service in various parts of your application, leading to problems and other unwanted and risky side-effects.
```

Comment:
Maybe something more vibrant/punchy/example-based/tangile.

For example - 
Otherwise: Poor Express` body-parser is tasked to parse 3MB JSON bodies in 1000 requests/sec rate. Your server is unlikely to perform anything else that parsing JSONs

Reply:
Looking into this 👍